doc-source/xray-sdk-dotnet-messagehandler.md

```diff
@@ -50,23 +50,41 @@ namespace SampleEBWebApplication
 
 ## Instrumenting incoming requests \(\.NET Core\)<a name="xray-sdk-dotnet-messagehandler-startupcs"></a>
 
-To instrument requests served by your application, call the `UseExceptionHandler`, `UseXRay`, and `UseStaticFiles` methods in the `Configure` method of your `Startup` class\.
+To instrument requests served by your application, call `UseXRay` method before any other middleware in the `Configure` method of your `Startup` class\.
```

Comment:
I think we can provide some reasoning to the customers for doing this. Ideally, the customers would want X-Ray to be the first to process the request and last to process the response in the ASP.Net request-response path. This will enable X-Ray to provide a complete picture of the customer's application and everything going on within it.

Reply:
Just updated